src/megha/Megha_Assignment24.java

```diff
@@ -0,0 +1,135 @@
+package megha;
+
+import java.util.Arrays;
+
+public class Megha_Assignment24 {
+	void findTotalWordsInStringWithLength(String str) {
+		String[] arr = str.split(" ");
+		System.out.println("Total Words in the given String are : " + arr.length);
+		for (int index = 0; index < arr.length; index++) {
+			int wordLength = arr[index].length();
+			System.out.println("Length of word " + arr[index] + " --> " + wordLength);
+		}
+	}
+
+	void findTotalVowelsInEachWordsInString(String str) {
+		String[] arr = str.split(" ");
+		for (int index = 0; index < arr.length; index++) {
+			int wordLength = arr[index].length();
+			String word = arr[index];
+			int wordVowel = 0;
+			for (int innerIndex = 0; innerIndex < wordLength; innerIndex++) {
+				char ch = word.charAt(innerIndex);
+				if (ch == 'a' || ch == 'A' || ch == 'e' || ch == 'E' || ch == 'i' || ch == 'I' || ch == 'o' || ch == 'O'
+						|| ch == 'u' || ch == 'U')
+					wordVowel++;
+			}
+			System.out.println("Vowels in the word " + word + " --> " + wordVowel);
```

Comment:
as you have to find the count then you can use uppercase() on string, so that it will reduce your || conditions.